packages/suite/src/components/guide/GuideButton.tsx

```diff
@@ -11,8 +11,7 @@ const Wrapper = styled.button<{ isGuideOpen?: boolean; isModalOpen?: boolean }>`
     justify-content: center;
     align-items: center;
     position: fixed;
-    z-index: ${({ isModalOpen }) =>
```

Comment:
I don't like that I can see the guide button over guide
https://user-images.githubusercontent.com/33235762/168983257-fc5be34e-af35-45d1-bf77-6a8d2a938f8f.mov

Reply:
This originally behaved differently with and without modal open. I made it consistent but went the wrong way.

Now that I added _position: relative_ to _GuidePanel_ and tweaked the z-indices, it should work as expected in both variants.